qmri_mtsat/sub-01/fmap/sub-01_flip-2_TB1DAM.json

```
@@ -0,0 +1,10 @@
+{
+  "IntendedFor": [
+    "fmap/sub-01_TB1map.nii.gz"
+  ],
```

Again,

```suggestion
```

or

```suggestion
  "IntendedFor": [
    "anat/sub-01_flip-2_mt-off_MTS.nii.gz"
  ],
```

or

```suggestion
  "IntendedFor": [
      "anat/sub-01_flip-1_mt-off_MTS.nii.gz",
      "anat/sub-01_flip-1_mt-on_MTS.nii.gz",
      "anat/sub-01_flip-2_mt-off_MTS.nii.gz"
  ],
```

if tools need this information to know that the output TB1Map is Intended for the output T1map.